apps/nowcasting-app/components/charts/solar-site-view/solar-site-chart.tsx

```diff
@@ -141,12 +217,32 @@ const SolarSiteChart: FC<{
   const legendItemContainerClasses = "flex flex-initial flex-col xl:flex-col justify-between";
   return (
     <div className={`flex flex-col flex-1 mb-1 ${className || ""}`}>
-      <div className="flex-auto mb-2">
+      <div className="flex-auto flex flex-col mb-2">
         <div className="flex content-between bg-ocf-gray-800 h-auto">
-          <div className="text-white lg:text-2xl md:text-lg text-base font-black m-auto ml-5 flex justify-evenly">
+          <div className="text-white lg:text-2xl md:text-lg text-base font-black m-auto mx-3 flex justify-evenly">
             All Sites
           </div>
-          <div className="flex justify-between flex-2 my-2 px-6">
+          <div className="flex-1 flex flex-col items-center justify-around text-2xs py-2 leading-none">
+            {Object.entries(
+              sitesLoadingState?.endpointStates || ({} as LoadingState<SitesEndpointStates>)
```

Comment:
Loading panel could be useful as a hover state. Could think about how this works with the "Loading" message.